contracts/colonyNetwork/ColonyNetwork.sol

```diff
@@ -545,12 +524,24 @@ contract ColonyNetwork is BasicMetaTransaction, ColonyNetworkStorage, Multicall
     }
   }
 
-  function isContract(address addr) internal returns (bool) {
-    uint256 size;
-    assembly { size := extcodesize(addr) }
-    return size > 0;
+  function toRootSkillId(uint256 _chainId) internal pure returns (uint256) {
+    return _chainId << 128;
+  }
+
+  function toChainId(uint256 _skillId) internal pure returns (uint256) {
```

Comment:
Similarly, `<adjective>skillIdToChainId`?

Reply:
I think it's fine as-is but if you feel strongly feel free to change it